cypress/locators/carousel/index.js

```diff
@@ -1,14 +1,20 @@
 import {
-  SLIDE_INDEX_SELECT, TRANSITION_SELECT, SLIDE, CAROUSEL, PREVIOUS_ARROW_BUTTON,
-  NEXT_ARROW_BUTTON, CAROUSEL_SLIDE_SELECTOR,
+  SLIDE_INDEX_SELECT, TRANSITION_SELECT, CLASSIC_SLIDE, CAROUSEL,
+  CAROUSEL_CONTENT, PREVIOUS_ARROW_BUTTON, NEXT_ARROW_BUTTON, CAROUSEL_SLIDE_SELECTOR,
 } from './locators';
 
 // knobs locators
 export const slideIndexSelect = () => cy.get(SLIDE_INDEX_SELECT);
 export const transitionSelect = () => cy.get(TRANSITION_SELECT);
 
 // component preview locators
-export const slide = () => cy.iFrame(SLIDE);
+export const classicSlide = () => cy.iFrame(CLASSIC_SLIDE);
+export const slide = i => cy.iFrame(CAROUSEL_CONTENT)
+  .find('div:nth-child(2)')
+  .find(`div[id="${i}"]`);
+export const clickableSlide = () => cy.iFrame(CAROUSEL_CONTENT)
```

Comment:
I don't think you need `clickableSlide`. It's the same as `slide`. 

Try use `slide` in you `When` step. You need to pass there "1" -> slide(1).click();

Reply:
Removed `clickableSlide` from this file and referred to `slide(1)` in the step 👍